static/app/components/pagination/paginationCaption.tsx

```diff
@@ -1,19 +0,0 @@
-import styled from '@emotion/styled';
```

Comment:
There wasn't really any reason for this to be separate from the Pagination component 🤷

Reply:
Which design? It looks like the captions always look the same. At least, this change didn't visually change anything (apart from a minor color change)

Reply:
I tried but couldn't find it anymore, but it was a Figma design created by Robin. Anyway, we must maintain consistency, so I don't see any problem with your changes